xbmc/rendering/gl/RenderSystemGL.cpp

```diff
@@ -345,8 +345,7 @@ void CRenderSystemGL::CaptureStateBlock()
   glMatrixTexture.Push();
 
   glDisable(GL_SCISSOR_TEST); // fixes FBO corruption on Macs
-  if (glActiveTextureARB)
```

Comment:
this one's been bugging me, clang-tidy keeps removing it as not needed so I would love to see this go :)